Extensions/Leaderboards/JsExtension.js

```diff
@@ -111,6 +111,37 @@ module.exports = {
       .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
       .setFunctionName('gdjs.evtTools.leaderboards.formatPlayerName');
 
+    extension
+      .addAction(
+        'DisplayLeaderboard',
+        _('Display leaderboard'),
+        _('Display the specified leaderboard on top of the game. If a leaderboard was already displayed on top of the game, the new leaderboard will replace it.'),
+        _('Display leaderboard _PARAM1_'),
+        '',
+        'JsPlatform/Extensions/leaderboard.svg',
+        'JsPlatform/Extensions/leaderboard.svg'
+      )
+      .addCodeOnlyParameter('currentScene', '')
+      .addParameter('leaderboardId', 'Leaderboard', '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
+      .setFunctionName('gdjs.evtTools.leaderboards.displayLeaderboard');
+
+    extension
```

Comment:
I wonder if we also should have a condiiton "LeaderViewOpened" which means the iframe was created and is displayed on top of the game (even if it is still loading). This way we have:

- An action to open the leaderboard view.
- We have a condition to know if it's opened (useful to pause your game or do something in the background).
- We have a condition to check if it errored (so you can at least display "oops, try again later") and could not be displayed.
- We have an action to close the leaderboard (though I guess in the majority of cases, it's the player that will close it).